docs/tutorial/index.md

```diff
@@ -319,17 +294,17 @@ date-created: 3 hours ago
 last-connection: just now
 ```
 
-At this point you could go ahead and add further users and control their permissions. However, for the purpose of this tutorial, we will skip ahead and deploy our chat service!
+At this point you could add further users and control their permissions. However, for the purpose of this tutorial, we will skip ahead and deploy our chat service!
 
 ## Provision infrastructure and operate applications
 
 ```{figure} tutorial-provision-deploy.svg
    :alt: A user uses the client to talk to the controller to talk to the cloud and to Charmhub to provision infrastructure and to deploy and operate charmed applications.
 
-   _A user uses the client to talk to the controller to talk to the cloud and to Charmhub to provision infrastructure and to deploy and operate charmed applications. The way this works is that any application unit created by Juju contains a unit agent that is constantly checking state with the controller agent and progressing the unit's state by executing the charm code to install, configure, etc., applications._
+   _A user interacts with the client to reach the controller. The controller talks to the cloud and to Charmhub to provision infrastructure and to deploy charms. Next to a deployed charm there is always a Juju agent which is constantly checking state against the Juju controller and executes the deployed charm accordingly to install, configure, and otherwise manage applications._
```

Comment:
```suggestion
   _A user interacts with the client to reach the controller. The controller talks to the cloud and to Charmhub to provision infrastructure and to deploy charms. Next to a deployed charm there is always a Juju agent which is periodically checking its internal state against the Juju controller and executes the deployed charm accordingly to install, configure, and otherwise manage applications._
```

Reply:
Fixed, thanks!